mne/io/base.py

```diff
@@ -1539,12 +1539,13 @@ def plot(self, events=None, duration=10.0, start=0.0, n_channels=20,
     def plot_psd(self, tmin=0.0, tmax=60.0, fmin=0, fmax=np.inf,
                  proj=False, n_fft=2048, picks=None, ax=None,
                  color='black', area_mode='std', area_alpha=0.33,
-                 n_overlap=0, dB=True, show=True, n_jobs=1, verbose=None):
+                 n_overlap=0, dB=True, average=False, show=True,
+                 n_jobs=1, verbose=None):
```

Comment:
we should probably add a `line_alpha` argument, and possibly have it default to `None`, which means `1.` for `average=True` and `0.1` or `0.25` for `average=False`, otherwise the plots will be too dense